WebRoot/js/zimbraMail/core/ZmMimeTable.js

```diff
@@ -60,6 +60,10 @@ ZmMimeTable.APP_XML     			= "application/xml";
 ZmMimeTable.APP_ZIMBRA_DOC			= "application/x-zimbra-doc";
 ZmMimeTable.APP_ZIP					= "application/zip";
 ZmMimeTable.APP_ZIP2				= "application/x-zip-compressed";
+ZmMimeTable.APP_RTF					= "application/rtf";
+ZmMimeTable.APP_OPENDOC_ODT			= "application/vnd.oasis.opendocument.text";
+ZmMimeTable.APP_OPENDOC_ODP 		= "application/vnd.oasis.opendocument.presentation";
+ZmMimeTable.APP_OPENDOC_ODS 		= "application/vnd.oasis.opendocument.spreadsheet";
```

Comment:
just in case, don't we need to add them to `ZmMimeTable._table` around line 118?

Reply:
Thanks for pointing out this.
I have added entry of this application types.